homeassistant/helpers/deprecation.py

```diff
@@ -143,3 +143,9 @@ def _print_deprecation_warning(obj: Any, replacement: str, description: str) ->
             description,
             replacement,
         )
+
+
+def warn_deprecated(logger: logging.Logger, msg: str, *args: Any) -> None:
+    """Log a deprecation warning message."""
+
+    logger.warning(msg, *args)
```

Comment:
This his just a proxy, I'm missing the point?

Reply:
True, as per discord chat will be useful to track what should be deprecated and when.

Simone

Reply:
> what should be deprecated 

We have different markers for that. I rather see it being collected in a generic deprecation helper package or something similar, document it and enforce it that way.

This feel like a loose fodder to me?

> and when.

Well, this doesn't tell anything about when?

Reply:
> Well, this doesn't tell anything about when?

Well, I wanted to add a `version: str` param, but others member not so I went that way even if it's not my preferred one ;-)